ScratchWikiSkinTemplate.php

```diff
@@ -271,7 +291,10 @@ function mod(el) {
 			if (!e) e = window.event;
 			if (!e.toElement) e.toElement = e.relatedTarget;
 			if (!e.toElement) return;
-			if (!btn.parentElement.contains(e.toElement)) {
+			if (
+				!btn.parentElement.contains(e.toElement)
+				&& !document.querySelector('.suggestions')?.contains(e.toElement)
```

Comment:
What is this doing?

Reply:
Handles an edge case where mousing out of the search bar into the search suggestion box hit a pixel between the search bar and the suggestions box, leading to the box closing as you moused to it. At least, I think.